docs/01-about/index.md

```diff
@@ -1,37 +1,58 @@
 ---
-title: About Nordcraft
-description: Learn to build powerful web applications with Nordcraft through extensive documentation, step-by-step guides and practical examples. Explore components, styling, APIs and advanced features or get assistance from Nordcraft's AI.
+title: What is Nordcraft?
+description: Welcome to the Nordcraft documentation. Learn how to build powerful web applications with Nordcraft through step-by-step guides and practical examples. Explore components, styling, APIs, and advanced features, or get help from Nordcraft's AI assistant.
```

Comment:
The AI assistant is not ready yet on the docs FYI

Reply:
I have removed this and we can add it in when it is ready.